packages/nc-gui/components/nc/SelectOption.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <a-select-option :value="value">
+    <div class="flex items-center w-full justify-between w-full gap-2">
+      <div class="truncate flex-1">
+        <slot />
+      </div>
+      <component v-if="isSelected" id="nc-selected-item-icon" class="text-primary w-6 h-6" :is="iconMap.check" />
+    </div>
+  </a-select-option>
+</template>
+<script lang="ts" setup>
+interface Props {
+  isSelected?: boolean
+  value?: string
+}
+
+const props = defineProps<Props>()
+const isSelected = computed(() => props.isSelected)
+const value = computed(() => props.value)
+console.log(props.isSelected, isSelected)
```

Comment:
remove that

Reply:
**REMOVED in BELOW COMMIT**

[fix : removed SelectOption component](https://github.com/nocodb/nocodb/pull/7181/commits/0ef580d48e9c98ce3c4cc2e3b4cdfd37f9637860)
[0ef580d](https://github.com/nocodb/nocodb/pull/7181/commits/0ef580d48e9c98ce3c4cc2e3b4cdfd37f9637860)